rpc/provider_test.go

```diff
@@ -101,7 +101,6 @@ func beforeEach(t *testing.T) *testConfiguration {
 		return &testConfig
 	}
 
-	testConfig.base = "https://free-rpc.nethermind.io/sepolia-juno"
```

Comment:
This line was overriding the default base URLs coming from testConfig and via the "env" flag in the CLI.
Now run `go test -v . -env ENVIROMENT` really get the corrent URL from the config, no needing to edit the "env.template" file if the URL already exists on the testConfig variable.